### ISSUE TITLE
@defaultValue with an array as a default value

### ISSUE THREAD

**Issue by mcohickVillanova** [microsoft/tsdoc#333]
Is it possible to have an array after the @defaultValue tag?

**Comment by octogonz** [comment]
Yes, it is a documentation block so you could put an English sentence as well

**Comment by Gerrit0** [comment]
Worth linking the doc page: https://tsdoc.org/pages/tags/defaultvalue/
